test2code/src/main/kotlin/com/epam/drill/plugins/test2code/Plugin.kt

```diff
@@ -246,6 +202,22 @@ class Plugin(
         }
         logger.info { "Scanned $classCount classes" }
     }
+
+    //TODO remove after admin refactoring
+    private fun createSession(
+        sessionId: String,
+        isRealtime: Boolean = true, isGlobal: Boolean = false
+    ) {
+        if (sessions[sessionId] != null)
+            return
+        synchronized(sessionId.intern()) {
+            if (sessions[sessionId] != null)
+                return
+            sendMessage(SessionStarted(sessionId, "AUTO", isRealtime, isGlobal, currentTimeMillis()))
+            logger.info { "Session $sessionId was created." }
+            sessions[sessionId] = true
```

Comment:
1.we keep assumption that session stays open
2.we'll get rid of session handling entirely in later java-agent version